examples/UrbanSprawl_figure_03.lua

```diff
@@ -0,0 +1,37 @@
+
+--@example Relationship among urban sprawl indicators and segregation indicators.
+
+import("urban")
+local dataFigA = {}
+local dataFigB = {}
+
+sessionInfo().graphics = false
+for meanIncome = 700, 1700, 100 do
```

Comment:
Use package calibration to execute multiple runs. It already returns a DataFrame with the output of the simulations, and executes `sessionInfo().graphics = true/false` automatically. For example, see

http://www.terrame.org/packages/doc/calibration/examples/daisy.lua

and

http://www.terrame.org/packages/doc/calibration/examples/fire-average.lua